docs/getting-started-guides/minikube.md

```diff
@@ -5,8 +5,7 @@ reviewers:
 - aaron-prindle
 title: Running Kubernetes Locally via Minikube
 ---
-
-Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs a single-node Kubernetes cluster inside a VM on your laptop for users looking to try out Kubernetes or develop with it day-to-day.
+If you want to try Kubernetes or develop by using Kubernetes on a day-to-day basis,then install minikube. You can run Kubernetes locally by using minikube. Minikube runs a single-node Kubernetes cluster inside a VM on your laptop.
```

Comment:
I don't think using Kubernetes on a day-to-day basis is a reason to install Minikube. I think the reason has more to do with experimenting in a local envrionment.